src/components/agencies/view-agency/view-agency.component.js

```diff
@@ -60,39 +79,32 @@ class ViewAgency extends Component {
             {loading ? (
               t("LOADING.LOADING")
             ) : (
-              <Fragment>
-                <div className="blue-color font-16">
-                  {t("NAVIGATION.AGENCIES")}
-                </div>
-                <div className="flex-row align-center">
-                  <div className="agency-name">{agency.name}</div>
-                  <div className={`status-icon ${status}-status-icon`}>
-                    {status}
+                <Fragment>
+                  <div className="item-label">{t("TABLE.AGENCY")}</div>
+                  <div className="item-name">{agencyInfo.agency}</div>
+                  <div className="font-16">{agencyInfo.description}</div>
+                  <div className="flex-row agency-box">
+                    <div className="agency-box-img">
+                      <img
+                        className="icon"
+                        src={require("../../../assets/site-icon.png")}
+                        alt="no logo"
+                      />
+                    </div>
+                    {agencyInfo.site}
                   </div>
-                </div>
-                <div className="font-16">{agency.description}</div>
-                <div className="flex-row agency-box">
-                  <div className="agency-box-img">
-                    <img
-                      className="icon"
-                      src={require("../../../assets/site-icon.png")}
-                      alt="no logo"
-                    />
+                  <div className="flex-row agency-box">
+                    <div className="agency-box-img">
+                      <img
+                        className="icon"
+                        src={require("../../../assets/email-icon.png")}
+                        alt="no logo"
+                      />
+                    </div>
+                    {agencyInfo.email}
                   </div>
-                  {agency.site}
-                </div>
-                <div className="flex-row agency-box">
-                  <div className="agency-box-img">
-                    <img
-                      className="icon"
-                      src={require("../../../assets/email-icon.png")}
-                      alt="no logo"
-                    />
-                  </div>
-                  {agency.email}
-                </div>
-              </Fragment>
-            )}
+                </Fragment>
+              )}
           </div>
           <NavLink to={EDIT_AGENCIES_PAGE.replace(":id", agency._id)}>
```

Comment:
This line is giving me an error when I run npm start locally on this branch:

Failed to compile.

./src/components/agencies/view-agency/view-agency.component.js
  Line 109:58:  'agency' is not defined  no-undef